xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamPVRManager.h

```diff
@@ -120,7 +120,7 @@ class CDVDInputStreamPVRManager
   bool m_isOtherStreamHack;
   PVR_STREAM_PROPERTIES *m_StreamProps;
   std::map<int, std::shared_ptr<CDemuxStream>> m_streamMap;
-  bool m_isRecording;
+  bool m_isRecording = false;
```

Comment:
This makes a difference if the class is used without calling CDVDInputStreamPVRManager::Open() and having it return true. I see no documentation warning of UB if this class is not used exactly as intended.